traveltimepy/dto/requests/time_map_geojson.py

```diff
@@ -62,10 +63,9 @@ class TimeMapRequestGeojson(TravelTimeRequest[FeatureCollection]):
     departure_searches: List[DepartureSearch]
     arrival_searches: List[ArrivalSearch]
 
-    def split_searches(self, window_size: int) -> List[FeatureCollection]:
+    def split_searches(self, window_size: int) -> List[TravelTimeRequest]:
```

Comment:
why did you replace FeatureCollection with TravelTimeRequest?

Reply:
Because that's what it is supposed to be. This method is used for the request, which doesn't change, regardless of what response format we choose. Therefore it should always be `TravelTimeRequest`. 

The only reason why my code worked before, was because I wasn't using this method at all (I accidentally got around using it, you can see it in the `send_post_geojson_async` method that I deleted).